Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -1871,7 +1871,7 @@ public unsafe void Actions_CanReadValueFromAction_InCallback_WithoutKnowingValue
             }
         };
 
-        InputSystem.QueueStateEvent(gamepad, new GamepadState {leftStick = new Vector2(0.123f, 0.234f)});
```

Comment:
Weird amount of formatting, is it our formatter got updated or something?

Reply:
I'm removing trailing whitespace in my editor and didn't notice this. But our formatter still allowed it.
I'll "put it back" as it was for consistency across the repo. Thanks for pointing it out 👍